packages/utilities/etc/utilities.api.md

```diff
@@ -1363,7 +1363,7 @@ export const useMergeStylesHooks: () => {
     useMergeStylesShadowRootContext: MergeStylesShadowRootContetHook;
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32492&commitId=efe0ab3d56b9380530a484abfdaa6c24c99a1b4a&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 22 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Bottom left edge.chromium.png|563| Changed|
|Callout.Bottom Right Edge RTL.chromium.png|566| Changed|
|Callout.Bottom right edge.chromium.png|563| Changed|
|Callout.Bottom Left Edge RTL.chromium.png|566| Changed|
|Callout.Bottom center.chromium.png|563| Changed|
|Callout.Bottom auto edge.chromium.png|563| Changed|
|Callout.Left top edge.chromium.png|563| Changed|
|Callout.Beak 25.chromium.png|563| Changed|
|Callout.Rendering callout attached to a rectangle.chromium.png|563| Changed|
|Callout.Right bottom edge.chromium.png|563| Changed|
|Callout.Gap space 25.chromium.png|563| Changed|
|Callout.Left center.chromium.png|563| Changed|
|Callout.No beak.chromium.png|563| Changed|
|Callout.Left bottom edge.chromium.png|563| Changed|
|Callout.No callout width specified.chromium.png|563| Changed|
|Callout.Right center.chromium.png|563| Changed|
|Callout.Right top edge.chromium.png|563| Changed|
|Callout.Top left edge.chromium.png|563| Changed|
|Callout.Root.chromium.png|563| Changed|
|Callout.Top auto edge.chromium.png|563| Changed|
|Callout.Top right edge.chromium.png|563| Changed|
|Callout.Top center.chromium.png|563| Changed|

</details>


  <details><summary><b>Checkbox</b> 13 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Checkbox.Checked disabled.chromium.png|431| Changed|
|Checkbox.Controlled Indeterminate.chromium.png|431| Changed|
|Checkbox.Checked.chromium.png|431| Changed|
|Checkbox.Controlled Indeterminate disabled.chromium.png|431| Changed|
|Checkbox.Custom render Checkbox.chromium.png|431| Changed|
|Checkbox.End RTL.chromium.png|435| Changed|
|Checkbox.End.chromium.png|431| Changed|
|Checkbox.Uncontrolled Indeterminate.chromium.png|431| Changed|
|Checkbox.Unchecked disabled.chromium.png|431| Changed|
|Checkbox.Unchecked.chromium.png|431| Changed|
|Checkbox.Uncontrolled Indeterminate disabled.chromium.png|431| Changed|
|Checkbox.Unchecked RTL.chromium.png|435| Changed|
|Checkbox.Multi-line Checkbox.chromium.png|431| Changed|

</details>


  <details><summary><b>Checkbox Indeterminate</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Checkbox Indeterminate.Controlled Indeterminate checkbox without onChange callback.chromium.png|431| Changed|
|Checkbox Indeterminate.Uncontrolled Indeterminate defaultChecked checkbox.chromium.png|431| Changed|
|Checkbox Indeterminate.Uncontrolled Indeterminate checkbox.chromium.png|431| Changed|
|Checkbox Indeterminate.Uncontrolled and Controlled Indeterminate checkbox.chromium.png|431| Changed|

</details>


  <details><summary><b>ChoiceGroup</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ChoiceGroup.Required.chromium.png|563| Changed|
|ChoiceGroup.With icons.chromium.png|563| Changed|
|ChoiceGroup.With Icons RTL.chromium.png|566| Changed|
|ChoiceGroup.Root.chromium.png|563| Changed|
|ChoiceGroup.With default size images.chromium.png|563| Changed|
|ChoiceGroup.With large size images.chromium.png|563| Changed|

</details>


  <details><summary><b>ColorPicker</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ColorPicker.Blue.chromium.png|563| Changed|
|ColorPicker.Root RTL.chromium.png|566| Changed|
|ColorPicker.Root.chromium.png|563| Changed|

</details>


  <details><summary><b>DocumentCard</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DocumentCard.Compact with preview image.chromium.png|563| Changed|
|DocumentCard.Not truncated.chromium.png|563| Changed|
|DocumentCard.With secondary title style.chromium.png|563| Changed|
|DocumentCard.Compact with preview list.chromium.png|563| Changed|

</details>


  <details><summary><b>FolderCover</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|FolderCover.Large Default Cover.chromium.png|563| Changed|
|FolderCover.Small Default Cover.chromium.png|563| Changed|
|FolderCover.Large Media Cover.chromium.png|563| Changed|
|FolderCover.Small Media Cover.chromium.png|563| Changed|
|FolderCover.Shared Cover.chromium.png|563| Changed|

</details>


  <details><summary><b>HoverCard</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|HoverCard.Root RTL.chromium.png|566| Changed|
|HoverCard.Root.chromium.png|563| Changed|

</details>


  <details><summary><b>Icon</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Icon.Image.chromium.png|563| Changed|
|Icon.Root.chromium.png|563| Changed|
|Icon.Color.chromium.png|563| Changed|

</details>


  <details><summary><b>Image</b> 21 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Image.Fit- CenterContain, image larger.chromium.png|563| Changed|
|Image.Fit- CenterContain, image smaller.chromium.png|563| Changed|
|Image.Fit- center, image smaller.chromium.png|563| Changed|
|Image.Fit- contain, image taller.chromium.png|563| Changed|
|Image.Fit- cover, image wider.chromium.png|563| Changed|
|Image.Fit- centerCover, image taller.chromium.png|563| Changed|
|Image.Fit- cover, image taller.chromium.png|563| Changed|
|Image.Fit- CenterContain, image taller.chromium.png|563| Changed|
|Image.Fit- centerCover, image larger.chromium.png|563| Changed|
|Image.Fit- CenterContain, image wider.chromium.png|563| Changed|
|Image.Fit- center, image larger.chromium.png|563| Changed|
|Image.Fit- contain, image wider.chromium.png|563| Changed|
|Image.Fit- centerCover, image smaller.chromium.png|563| Changed|
|Image.Fit- none, image larger.chromium.png|563| Changed|
|Image.Fit- none, image smaller.chromium.png|563| Changed|
|Image.Fit- centerCover, image wider.chromium.png|563| Changed|
|Image.Maximize frame, landscape container.chromium.png|563| Changed|
|Image.Maximize frame, portrait container.chromium.png|563| Changed|
|Image.No fit, only height.chromium.png|563| Changed|
|Image.No fit, only width.chromium.png|563| Changed|
|Image.No fit, no w-h.chromium.png|563| Changed|

</details>


  <details><summary><b>Keytip</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Keytip.Disabled.chromium.png|563| Changed|
|Keytip.Offset.chromium.png|563| Changed|
|Keytip.Root.chromium.png|563| Changed|

</details>


  <details><summary><b>Label</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Label.Root RTL.chromium.png|788| Changed|
|Label.Required.chromium.png|797| Changed|
|Label.Disabled.chromium.png|797| Changed|
|Label.Root.chromium.png|797| Changed|

</details>


  <details><summary><b>Layer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Layer.Root RTL.chromium.png|661| Changed|
|Layer.Root.chromium.png|664| Changed|

</details>


  <details><summary><b>Link</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Link.Disabled.chromium.png|964| Changed|
|Link.No Href Disabled.chromium.png|964| Changed|
|Link.No Href.chromium.png|964| Changed|
|Link.Root.chromium.png|964| Changed|
|Link.Root RTL.chromium.png|969| Changed|
|Link.Underlined Disabled.chromium.png|964| Changed|
|Link.Underlined.chromium.png|964| Changed|

</details>


  <details><summary><b>MediaTile</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|MediaTile.Media tile with single activity line.chromium.png|563| Changed|
|MediaTile.Media tile with nameplate hidden until hover.chromium.png|563| Changed|
|MediaTile.Media tile with two activity lines.chromium.png|563| Changed|
|MediaTile.Media tile with very long name and activity.chromium.png|563| Changed|

</details>


  <details><summary><b>Modal</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Modal.Modeless.chromium.png|431| Changed|
|Modal.Root.chromium.png|431| Changed|

</details>


  <details><summary><b>Overlay</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Overlay.Root.chromium.png|749| Changed|
|Overlay.Dark.chromium.png|749| Changed|
|Overlay.Root RTL.chromium.png|756| Changed|

</details>


  <details><summary><b>Persona</b> 16 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Persona.Size 100 Extra Large RTL.chromium.png|435| Changed|
|Persona.Initials RTL.chromium.png|435| Changed|
|Persona.Persona with children.chromium.png|431| Changed|
|Persona.size10 (tiny).chromium.png|431| Changed|
|Persona.size100 (extraLarge).chromium.png|431| Changed|
|Persona.Initials.chromium.png|431| Changed|
|Persona.Size 120 RTL.chromium.png|435| Changed|
|Persona.default size, details hidden.chromium.png|431| Changed|
|Persona.default size, presences.chromium.png|431| Changed|
|Persona.size32.chromium.png|431| Changed|
|Persona.size48 (regular).chromium.png|431| Changed|
|Persona.size120.chromium.png|431| Changed|
|Persona.size24 (extraExtraSmall).chromium.png|431| Changed|
|Persona.size28 (extraSmall).chromium.png|431| Changed|
|Persona.size40 (small).chromium.png|431| Changed|
|Persona.size72 (large).chromium.png|431| Changed|

</details>


  <details><summary><b>ProgressIndicator</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ProgressIndicator.0%.chromium.png|721| Changed|
|ProgressIndicator.100%.chromium.png|721| Changed|
|ProgressIndicator.50%.chromium.png|721| Changed|
|ProgressIndicator.50 RTL.chromium.png|721| Changed|

</details>


  <details><summary><b>ScrollablePane</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ScrollablePane.Default ScrollablePane Example.chromium.png|1043| Changed|

</details>


  <details><summary><b>Separator</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Separator.Horizontal End Aligned RTL.chromium.png|898| Changed|
|Separator.Horizontal Start Aligned.chromium.png|896| Changed|
|Separator.Horizontal Start Aligned RTL.chromium.png|898| Changed|
|Separator.Vertical Center Aligned.chromium.png|896| Changed|
|Separator.Horizontal End Aligned.chromium.png|896| Changed|
|Separator.Root RTL.chromium.png|898| Changed|
|Separator.Vertical Start Aligned RTL.chromium.png|898| Changed|
|Separator.Root.chromium.png|896| Changed|
|Separator.Vertical Center Aligned RTL.chromium.png|898| Changed|
|Separator.Vertical End Aligned RTL.chromium.png|898| Changed|
|Separator.Vertical End Aligned.chromium.png|896| Changed|
|Separator.Vertical Start Aligned.chromium.png|896| Changed|

</details>


  <details><summary><b>Shimmer</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Shimmer.50% width.chromium.png|669| Changed|
|Shimmer.Circle Gap Line RTL.chromium.png|665| Changed|
|Shimmer.Basic.chromium.png|669| Changed|
|Shimmer.Circle Gap Line.chromium.png|669| Changed|
|Shimmer.Custom elements.chromium.png|669| Changed|
|Shimmer.Data loaded.chromium.png|669| Changed|
|Shimmer.Data not loaded.chromium.png|669| Changed|
|Shimmer.Custom elements on themed background.chromium.png|669| Changed|

</details>


  <details><summary><b>Signals</b> 21 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Signals.Blocked.chromium.png|563| Changed|
|Signals.Awaiting approval.chromium.png|563| Changed|
|Signals.Comments.chromium.png|563| Changed|
|Signals.Emailed.chromium.png|563| Changed|
|Signals.Comments (count).chromium.png|563| Changed|
|Signals.New.chromium.png|563| Changed|
|Signals.Follow.chromium.png|563| Changed|
|Signals.Malware detected.chromium.png|563| Changed|
|Signals.NotFollow.chromium.png|563| Changed|
|Signals.Mention.chromium.png|563| Changed|
|Signals.Read-only.chromium.png|563| Changed|
|Signals.Missing metadata.chromium.png|563| Changed|
|Signals.New (positioning).chromium.png|563| Changed|
|Signals.Record.chromium.png|563| Changed|
|Signals.Someone checked out.chromium.png|563| Changed|
|Signals.Shared.chromium.png|563| Changed|
|Signals.Warning.chromium.png|563| Changed|
|Signals.Trending.chromium.png|563| Changed|
|Signals.Unseen reply.chromium.png|563| Changed|
|Signals.Unseen edit.chromium.png|563| Changed|
|Signals.You checked out.chromium.png|563| Changed|

</details>


  <details><summary><b>Slider</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Slider.EqualMinMax.chromium.png|431| Changed|
|Slider.Disabled.chromium.png|431| Changed|
|Slider.Step less than 1.chromium.png|431| Changed|
|Slider.Root RTL.chromium.png|435| Changed|
|Slider.Max not multiple of step.chromium.png|431| Changed|
|Slider.Root.chromium.png|431| Changed|
|Slider.Vertical.chromium.png|431| Changed|

</details>


  <details><summary><b>Spinner</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Spinner.Extra small.chromium.png|873| Changed|
|Spinner.Label at top.chromium.png|873| Changed|
|Spinner.Label.chromium.png|873| Changed|
|Spinner.Medium.chromium.png|873| Changed|
|Spinner.Label on the right.chromium.png|873| Changed|
|Spinner.Large.chromium.png|873| Changed|
|Spinner.Label on the left.chromium.png|873| Changed|
|Spinner.Small.chromium.png|873| Changed|

</details>


  <details><summary><b>Stack</b> 44 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Stack.Horizontal Stack - Bottom-aligned.chromium.png|912| Changed|
|Stack.Horizontal Stack - Gap.chromium.png|912| Changed|
|Stack.Horizontal Stack - Growing items.chromium.png|740| Changed|
|Stack.Horizontal Stack - Padding.chromium.png|912| Changed|
|Stack.Horizontal Stack - Default.chromium.png|912| Changed|
|Stack.Horizontal Stack - Box shadow around items.chromium.png|912| Changed|
|Stack.Horizontal Stack - Right-aligned.chromium.png|912| Changed|
|Stack.Horizontal Stack - Item alignments.chromium.png|740| Changed|
|Stack.Horizontal Stack - Horizontally centered.chromium.png|912| Changed|
|Stack.Horizontal Stack - Reversed.chromium.png|912| Changed|
|Stack.Horizontal Stack - Space around.chromium.png|912| Changed|
|Stack.Horizontal Stack - Shrinking items.chromium.png|740| Changed|
|Stack.Horizontal Stack - Wrap with specified vertical gap.chromium.png|912| Changed|
|Stack.Horizontal Stack - Space evenly.chromium.png|912| Changed|
|Stack.Horizontal Stack - Vertically centered.chromium.png|912| Changed|
|Stack.Horizontal Stack - Space between.chromium.png|912| Changed|
|Stack.Horizontal Stack Gap RTL.chromium.png|916| Changed|
|Stack.Horizontal Stack - Wrap.chromium.png|912| Changed|
|Stack.Vertical Stack - Gap.chromium.png|912| Changed|
|Stack.Horizontal Stack Item Alignments RTL.chromium.png|739| Changed|
|Stack.Horizontal Stack Wrap RTL.chromium.png|916| Changed|
|Stack.Horizontal Stack Right Aligned RTL.chromium.png|916| Changed|
|Stack.Horizontal Stack Default RTL.chromium.png|916| Changed|
|Stack.Horizontal Stack Reversed RTL.chromium.png|916| Changed|
|Stack.Vertical Stack - Growing items.chromium.png|740| Changed|
|Stack.Vertical Stack - Bottom-aligned.chromium.png|912| Changed|
|Stack.Vertical Stack - Default.chromium.png|912| Changed|
|Stack.Vertical Stack - Box shadow around items.chromium.png|912| Changed|
|Stack.Vertical Stack - Horizontally centered.chromium.png|912| Changed|
|Stack.Vertical Stack - Padding.chromium.png|912| Changed|
|Stack.Vertical Stack - Item alignments.chromium.png|740| Changed|
|Stack.Vertical Stack - Space evenly.chromium.png|912| Changed|
|Stack.Vertical Stack - Space around.chromium.png|912| Changed|
|Stack.Vertical Stack - Reversed.chromium.png|912| Changed|
|Stack.Vertical Stack - Space between.chromium.png|912| Changed|
|Stack.Vertical Stack - Shrinking items.chromium.png|740| Changed|
|Stack.Vertical Stack - Right-aligned.chromium.png|912| Changed|
|Stack.Vertical Stack - Wrap.chromium.png|912| Changed|
|Stack.Vertical Stack - Vertically centered.chromium.png|912| Changed|
|Stack.Vertical Stack Wrap RTL.chromium.png|916| Changed|
|Stack.Vertical Stack Default RTL.chromium.png|916| Changed|
|Stack.Vertical Stack Right Aligned RTL.chromium.png|916| Changed|
|Stack.Vertical Stack Item Alignments RTL.chromium.png|739| Changed|
|Stack.Vertical Stack Reversed RTL.chromium.png|916| Changed|

</details>


  <details><summary><b>TextField</b> 20 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TextField.Icon.chromium.png|431| Changed|
|TextField.Multiline RTL.chromium.png|435| Changed|
|TextField.Error.chromium.png|431| Changed|
|TextField.Icon RTL.chromium.png|435| Changed|
|TextField.Borderless.chromium.png|431| Changed|
|TextField.Error RTL.chromium.png|435| Changed|
|TextField.Disabled.chromium.png|431| Changed|
|TextField.Multiline nonresizable.chromium.png|431| Changed|
|TextField.Multiline.chromium.png|431| Changed|
|TextField.Placeholder RTL.chromium.png|435| Changed|
|TextField.Placeholder.chromium.png|431| Changed|
|TextField.Prefix With Value Disabled And Prefix Style Override RTL.chromium.png|435| Changed|
|TextField.Prefix With Value Disabled RTL.chromium.png|435| Changed|
|TextField.Prefix with Value, Disabled, and Prefix Style-Override.chromium.png|431| Changed|
|TextField.Required.chromium.png|431| Changed|
|TextField.Prefix with Value, Disabled.chromium.png|431| Changed|
|TextField.Root.chromium.png|431| Changed|
|TextField.Suffix.chromium.png|431| Changed|
|TextField.Underlined.chromium.png|431| Changed|
|TextField.Suffix RTL.chromium.png|435| Changed|

</details>


  <details><summary><b>Tile</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tile.Tile with no content and long text.chromium.png|563| Changed|
|Tile.Document tile with fit landscape image.chromium.png|563| Changed|
|Tile.Document tile with fit portrait image.chromium.png|563| Changed|
|Tile.Document tile with icon-sized image.chromium.png|563| Changed|
|Tile.Document tile with icon.chromium.png|563| Changed|

</details>


  <details><summary><b>Toggle</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Toggle.Checked RTL.chromium.png|929| Changed|
|Toggle.Checked.chromium.png|933| Changed|
|Toggle.Disabled checked.chromium.png|933| Changed|
|Toggle.With inline label (JSX Element).chromium.png|933| Changed|
|Toggle.Unchecked RTL.chromium.png|929| Changed|
|Toggle.With inline label and without onText and offText.chromium.png|933| Changed|
|Toggle.With inline label (JSX Element) and without onText and offText.chromium.png|933| Changed|
|Toggle.Unchecked.chromium.png|933| Changed|
|Toggle.Disabled unchecked.chromium.png|933| Changed|
|Toggle.With inline label.chromium.png|933| Changed|
|Toggle.With inline label disabled.chromium.png|933| Changed|
|Toggle.With inline label disabled and without onText and offText.chromium.png|933| Changed|

</details>


  <details><summary><b>Tooltip</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tooltip.Default.chromium.png|563| Changed|

</details>


  <details><summary><b>Tooltip - Multiple</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tooltip - Multiple.Two Tooltips.chromium.png|563| Changed|

</details>


  <details><summary><b>react-charting-HeatMapChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-HeatMapChart.Basic RTL.chromium.png|372| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.chromium.png|1| Changed|

</details>


  <details><summary><b>react-charting-VerticalBarChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-VerticalBarChart.Basic - Secondary Y Axis.chromium.png|4| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>